driver.go

```diff
@@ -295,21 +297,23 @@ func mount(accountName, accountKey, mountpoint string, options VolumeOptions) er
 		options.GID = "0"
 	}
 	mount := fmt.Sprintf("//%s.file.core.windows.net/%s", accountName, options.Share)
-	opts := fmt.Sprintf(
-		"vers=3.0,username=%s,password=%s,dir_mode=%s,file_mode=%s,uid=%s,gid=%s",
-		accountName,
-		accountKey,
-		options.DirMode,
-		options.FileMode,
-		options.UID,
-		options.GID,
-	)
+	opts = append(opts, "vers=3.0")
```

Comment:
> As a side note, I've tested this and seems to be working reliably so far.

@jgreat sounds great , thank you!



> I can format the "required" options like your above comment, but any optional or boolean ones I'd still do as an append to the slice.

yep you only append `nolock` anyway. let's just append that.

> For mount, boolean options either exist or are left off.

ok we can leave it as is.

happy to merge once this is fixed.